client/src/css/tabs.css

```diff
@@ -100,8 +97,16 @@ body, #app, #app>div{
   align-items: center;
   justify-content: center;
 }
-  
-.gridContainer > div > p{
-  font-size: 14px;
+
+.editButton{
+  position: absolute !important;
+  top: 20px;
+  right: 20px;
```

Comment:
Since these values are hardcoded, does it still look okay on other screen sizes?

Reply:
Not vital, but I can imagine STE would use an ipad or something similar instead of a laptop